addons/sms/wizard/sms_composer_views.xml

```diff
@@ -21,13 +21,13 @@
                         <field name="sanitized_numbers" invisible="1"/>
 
                         <!-- Single mode information (invalid number) -->
-                        <div colspan="2" class="alert alert-danger text-center mb-3" role="alert"
+                        <div colspan="2" class="alert alert-danger text-center mb-3 mt-n4" role="alert"
```

Comment:
Have you tried moving that warning the the header section of the form?
I'm not a fan of using negative margin, seems very hackish.

Reply:
Hello @awa-odoo 

I tried with the header section but the header is not available in the new window. 
Please give suggestions for a negative margin.

Thanks!

Reply:
Hello @mish-odoo 

Then you can put those warnings just after the `<sheet>` tag, outside of any `<group>`.
It looks very slightly different from your version but it's better technically speaking.

(Also, you can add a space at the end of the 2 first spans, otherwise it looks like this:
"Invalid number, make sure to set a country on theContactorto specify the country code.")

Cheers!